src/pages/AddLiquidity/index.tsx

```diff
@@ -459,6 +461,16 @@ export default function AddLiquidity({
   const showApprovalB =
     !argentWalletContract && approvalB !== ApprovalState.APPROVED && !!parsedAmounts[Field.CURRENCY_B]
 
+  const pendingText = mustCreateSeparately
+    ? `Creating ${currencies[Field.CURRENCY_A]?.symbol}/${currencies[Field.CURRENCY_B]?.symbol} ${
```

Comment:
need a translation macro

Reply:
will do in a follow-up. There's a few transaction texts like this one that needs disentangling